Update pim-how-to-activate-role.yml

danwdavey opened the pull request:
I have just encountered a customer who has this control enabled and the guidance does not match their experience. Therefore added a note that if customer are using the other control in Entra, then they won't be able to activate their PIM Roles as they have no assigned 'standing' roles

danwdavey committed:
Update pim-how-to-activate-role.yml

v-regandowner commented on docs/id-governance/privileged-identity-management/pim-how-to-activate-role.yml:
```diff
@@ -30,7 +30,8 @@ introduction: |
   >[!IMPORTANT]
   >If a user activating an administrative role is signed in to Microsoft Teams on a mobile device, they will receive a notification from the Teams app saying "Open Teams to continue receiving notifications for *&lt;email address&gt;*", or "*&lt;email address&gt;* needs to sign in to see notifications". The user will need to open the Teams app to continue receiving notifications. This behavior is  ...
-
+  >[!IMPORTANT]
```
```suggestion

 >[!IMPORTANT]
```

v-regandowner replied:
This change should clear the build error.

ktoliver committed:
Update PIM role activation instructions

Clarified notification behavior for Teams users activating roles and emphasized access restrictions for eligible identities.
